src/python_testing/TC_OpstateCommon.py

```diff
@@ -1259,11 +1270,12 @@ async def TEST_TC_OPSTATE_BASE_2_6(self, endpoint=1):
         sub_handler = ClusterAttributeChangeAccumulator(cluster)
         await sub_handler.start(self.default_controller, self.dut_node_id, endpoint)
 
+        self.step(3)
         if self.pics_guard(self.check_pics(f"{self.test_info.pics_code}.S.M.ST_RUNNING")):
-            self.step(3)
             self.send_manual_or_pipe_command(name="OperationalStateChange",
                                              device=self.device,
-                                             operation="Start")
+                                             operation="Start",
+                                             msg="Put the DUT into the running state")
```

Comment:
Question about the statement that follows below (i.e. time.sleep(1)): should we add a PIXIT code to control these waits between issuing a command to the DUT and reading some attribute to check the effect? For this TC and for many others. The "real DUTs", that the vendors will attempt to certify, may have wild variations for the response times.

Not something to be fixed in this PR, obviously, but if you agree, or have other ideas, would you mind creating an issue for this?

Reply:
I think it's fine - the test is just attempting to verify that the device isn't over reporting on the Q attribute - 10 s should be fine to assess that.